### PR TITLE
WebGL device ignores WebGL1 request and always creates WebGL2 device

### PR DESCRIPTION
**Pull request by mvaligursky**
- WebGL device ignores WebGL1 request and always creates WebGL2 device

### CODE REVIEW

**Comment by kpal81xd** [comment]
Prob worth still add a description?

**Review by kpal81xd — APPROVED** [review]

File: src/platform/graphics/webgl/webgl-graphics-device.js [hunk 1]
```diff
@@ -380,8 +377,7 @@ class WebglGraphicsDevice extends GraphicsDevice {
         if (options.gl) {
             gl = options.gl;
         } else {
-            const preferWebGl2 = (options.preferWebGl2 !== undefined) ? options.preferWebGl2 : true;
-            const names = preferWebGl2 ? ["webgl2", "webgl", "experimental-webgl"] : ["webgl", "experimental-webgl"];
+            const names = ["webgl2", "webgl", "experimental-webgl"];
```
Comment (kpal81xd) [inline src/platform/graphics/webgl/webgl-graphics-device.js:380]: So there will still be an internal fallback to webgl1 if webgl2 fails?

File: examples/src/examples/misc/multi-app/example.mjs [hunk 2]
```diff
@@ -21,9 +21,7 @@ const createApp = async function (deviceType) {
         device = new pc.WebgpuGraphicsDevice(canvas, {});
         await device.initWebGpu(rootPath + '/static/lib/glslang/glslang.js', rootPath + '/static/lib/twgsl/twgsl.js');
     } else if (deviceType === 'webgl1' || deviceType === 'webgl2') {
-        device = new pc.WebglGraphicsDevice(canvas, {
```
Comment (yaustar) [inline examples/src/examples/misc/multi-app/example.mjs:24]: Do we want to add a warning if `webgl1` is requested saying deprecated or something?

Comment (mvaligursky) [reply examples/src/examples/misc/multi-app/example.mjs:24]: No really, we'll string it out from examples completely so it cannot be requested by the examples

**Pull request merged**